frappe/www/update-password.html

```diff
@@ -79,11 +79,11 @@
 					strength_indicator.addClass('hidden');
 					strength_message.addClass('hidden');
 					$('.page-card-head .indicator')
-						.removeClass().addClass('indicator green')
-						.html("{{ _('Password Updated') }}");
+						.removeClass().addClass('indicator blue')
+						.html("{{ _('Status Updated') }}");
 					if(r.message) {
 						frappe.msgprint({
-							message: "{{ _("Password Updated") }}",
+							message: `{{ _('${r.message}') }}`,
```

Comment:
Can we make this more explicit? We can have a success and failure response argument that can be sent or perhaps using the http response code. based on this we can format the msgprint with title and indicator.

Reply:
This won't work, as the translation API expects the string to be embedded in the `_('')` method.